Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -2809,7 +2814,9 @@ void SoftwareGLContext::gl_tex_gen_floatv(GLenum coord, GLenum pname, GLfloat co
         texture_coordinate_generation(capability).object_plane_coefficients = { params[0], params[1], params[2], params[3] };
         break;
     case GL_EYE_PLANE: {
-        auto inverted_model_view_matrix = m_model_view_matrix.inverse();
+        auto inverse_model_view_matrix = m_model_view_matrix.inverse();
+        RETURN_WITH_ERROR_IF(inverse_model_view_matrix.is_error(), GL_INVALID_OPERATION);
```

Comment:
Might want to leave a comment here that this is off spec/docs, as the documentation doesn't state this will cause `GL_INVALID_OPERATION`

Reply:
Well, I'm not even sure we should return `GL_INVALID_OPERATION`. What would be the expected behavior for `GL_EYE_PLANE` when the model view matrix is non-inversable?

Reply:
https://www.khronos.org/registry/OpenGL-Refpages/gl2.1/xhtml/glTexGen.xml

"M is the modelview matrix when glTexGen is invoked. If M is poorly conditioned or singular, texture coordinates generated by the resulting function may be inaccurate or undefined. "

Reply:
Alright cool. In that case let's work out a correct condition for this. @sunverwerth Did you work out whether or not nVidia return `NaN/INF`?

Reply:
Eh I was a bit too optimistic here. Since their drivers are closed this would be hard to verify :D